### PR TITLE
[feat] added sidebar and navbar

### PR DESCRIPTION
**Pull request by aryan-13**
### Overview

- Added nav component
- Added sidebar component 
- made them responsive

### Preview
image
Mobile:
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import './App.css';
+import Navbar from './Components/Header/Navbar';
```
Comment (anshikaa12) [inline src/App.js]: import from one index.js file for cleaner code

File: src/Components/Header/Sidebar.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
+.sidebar-container {
+	width: 100vw;
+	position: fixed;
+	bottom: 0;
+	padding: 1rem 0px;
+	/* margin-bottom: 1rem; */
```
Comment (anshikaa12) [inline src/Components/Header/Sidebar.css:6]: remove comments

File: src/Components/Header/Sidebar.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	},
+];
+function Sidebar() {
+	return (
+		<div className="sidebar-container ">
+			<ul className="sidebar-list">
+				{sidebarItems.map((item) => {
+					return (
+						<li
+							className={
```
Comment (anshikaa12) [inline src/Components/Header/Sidebar.jsx:43]: use prettier to format

**Review by anshikaa12 — APPROVED** [review]
code is well structured
liked the folder structure

**Review by junaidshaikh-js — APPROVED** [review]
Good work! 

Suggestions made by @anshikaa12 can be done, everything else looks good.

**Pull request merged**